src/components/Slider/styles.css

```diff
@@ -0,0 +1,60 @@
+.thumb {
```

Comment:
please don't use `px` units, we use `rem` units all over the app.

we defined our root font for be 10px, so convert any px value to rem using `1rem = 10px`